config-connector/solutions/iam/kpt/pubsub-topic/README.md

```diff
@@ -0,0 +1,54 @@
+Pubsub Topic
+==================================================
+
+# NAME
+
+  pubsub-topic
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to grant a role to a particular member for a PubSub topic.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  BASE=https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+  kpt pkg get $BASE/config-connector/solutions/iam/kpt/pubsub-topic pubsub-topic
+  ```
+
+# REQUIREMENTS
+
+  A working cluster with Config Connector installed.
+  A GCP project with PubSub service enabled.
+
+# USAGE
+
+  Replace `${PROJECT_ID?}` with your desired project ID value:
+  ```
+  kpt cfg set . project-id your-project-id
+  ```
+
+  Replace `${IAM_NAME?}` with the GCP identity to grant access to:
+  ```
+  kpt cfg set . iam-name user:name@example.com
+  ```
+
+  Optionally set the name of the PubSub topic and the role to grant (defaults are allowed-topic and editor).
+  ```
+  kpt cfg set . topic-name your-topic
+  kpt cfg set . role viewer
```

Comment:
Can you make clear to the user what values for 'role' are acceptable here?

Reply:
The README now looks [like this](https://cloud.google.com/iam/docs/understanding-roles#pub-sub-roles). Do you think it's more confusing to do it this way? I think it might be nice to keep the substitution setter as-is, but I can drop it if we think it's easier to just have something like:
```
kpt cfg set . role pubsub.viewer
```
or
```
kpt cfg set . role roles/pubsub.viewer
```
Might also be better to rename the setter to `pubsub-role`, what do you think?

Reply:
Leave the setter name as `role` and use the fully qualified `roles/pubsub.viewer`. The idea to use a substitution setter was good, but not something we want to take any pains to explain/maintain.

Reply:
Made this change just now.